src/maker/api.rs

```diff
@@ -71,8 +71,6 @@ pub const MIN_CONTRACT_REACTION_TIME: u16 = 20;
 ///
 /// These parameters define the fees charged by Makers in a coinswap transaction.
 ///
-/// TODO: These parameters are currently hardcoded. Consider making them configurable for Makers in the future.
-///
 /// - `BASE_FEE`: A fixed base fee charged by the Maker for providing its services
 /// - `AMOUNT_RELATIVE_FEE_PCT`: A percentage fee based on the swap amount.
 /// - `TIME_RELATIVE_FEE_PCT`: A percentage fee based on the refund locktime (duration the Maker must wait for a refund).
```

Comment:
We should change this section accordingly now. Base fee and amount_pct_fee are not constants anymore.  We should mention their default values and that they are configurable in the maker config.